.github/workflows/generate-sdk-python.yaml

```diff
@@ -0,0 +1,123 @@
+---
+#################################################################################
+# Automatic pipeline that triggers the update of the SDK Python
+#################################################################################
+name: SDK Python
+
+on:
+  workflow_call:
+    inputs:
+      base_branch:
+        required: true
+        type: string
+
+
+concurrency:
+  group: sdk-python
+  cancel-in-progress: true
+
+jobs:
+  sdk-python:
+    name: SDK Python
+    runs-on: ubuntu-2-core
+    steps:
+      - name: Checkout hoprnet repository
+        uses: actions/checkout@v4
+        # with:
+        #   ref: ${{ inputs.base_branch }}
+
+      - name: Sccache Setup
+        uses: mozilla-actions/sccache-action@v0.0.3
+        with:
+          version: "v0.4.0"
+
+      - name: Install Nix
+        uses: cachix/install-nix-action@v24
+        with:
+          github_access_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Nix Cache
+        uses: cachix/cachix-action@v13
+        with:
+          name: hoprnet
+          authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+        continue-on-error: true
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "dir=$(pwd)/.yarn/cache" >> $GITHUB_OUTPUT
+
+      - name: Restore yarn cache
+        uses: actions/cache@v3
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Restore cargo cache
+        uses: actions/cache@v3
+        with:
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock', 'rust-toolchain.toml') }}
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
+            .cargo/.package-cache/
+
+      # Doing the checkout after restoring the cache
+      - name: Checkout hoprd-python-sdk repository
+        uses: actions/checkout@v4
+        with:
+          repository: hoprnet/hoprd-sdk-python
+          path: "./hoprd-sdk-python"
+          token: ${{ secrets.GH_RUNNER_TOKEN }}
+          ref: ${{ inputs.base_branch }}
+
+      - name: Setup
+        id: setup
+        run: |
+          export current_version=$(./scripts/get-current-version.sh docker)
+          echo "current_version=${current_version}" >> $GITHUB_OUTPUT
+          export open_api_spec="https://raw.githubusercontent.com/hoprnet/hoprnet/${{ inputs.base_branch }}/packages/hoprd/rest-api-v3-spec.yaml"
```

Comment:
This spec does not exist after the migration to Rust. It has to be generated by running the command `hoprd-api-spec`.

Reply:
Adapted to the new way of generating the open_api spec in fully rust native binary